modules/user_accounts/templates/form_edit_user.tpl

```diff
@@ -397,6 +397,7 @@
                 {/if}
             </div>
       {/if}
+      {if $form.Active|default}
```

Comment:
This changes was made because I noticed while displaying an admin account that an extra invisible div taking up space was being made for "Active" while Active isn't meant to be displayed for admin accounts.